src/deprecated.js

```diff
@@ -23,8 +23,10 @@ jQuery.fn.extend( {
 			this.off( selector, "**" ) :
 			this.off( types, selector || "**", fn );
 	}
+
```

Comment:
Looks good, though for cleanliness may want to remove this blank line which wasn't there before.
👍